deployment/updates/scripts/xml/updateCommentWasAddedToEntryEmailNotificationTemplate.xml

```diff
@@ -0,0 +1,125 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<xml>
+        <config>
+                <serviceUrl>{prompt:service URL:}/</serviceUrl>
+                <partnerId>-2</partnerId>
+                <clientTag>Stand alone php 1.0.0</clientTag>
+                <curlTimeout>120</curlTimeout>
+                <userAgent>Stand alone php 1.0.0</userAgent>
+        </config>
+        <session>
+                <!-- script will ask for username / password interactively -->
+        </session>
+
+        <multirequest>
+                <!-- Entry was added to channel -->
+                <request service="eventNotificationTemplate" action="listTemplates" plugin="eventNotification">
+                        <filter objectType="KalturaEventNotificationTemplateFilter">
+                                <partnerId>0</partnerId>
+                                <systemNameEqual>Comment_Was_Added_To_Entry</systemNameEqual>
+                        </filter>
+                        <pager objectType="KalturaFilterPager">
+                                <pageSize>1</pageSize>
+                        </pager>
+                </request>
+                <request service="eventNotificationTemplate" action="update" plugin="eventNotification" partnerId="0">
+                	<id>{1:result:objects:0:id}</id>
+                   	<template objectType="KalturaEmailNotificationTemplate">
+                                <body>A new comment was added to your video {entry_name}.Comment: {comment_text}. You can access the video page here: [AppEntryUrl]{entry_id}</body>
+                                <contentParameters objectType="array">
+ 					<item objectType="KalturaEventNotificationParameter">
+ 						<key>from_email</key>
+ 						<value objectType="KalturaEvalStringField">
+ 							<code>kConf::get("partner_notification_email")</code>
+ 						</value>
+ 					</item>
+ 					<item objectType="KalturaEventNotificationParameter">
+ 						<key>from_name</key>
+ 						<value objectType="KalturaEvalStringField">
+ 							<code>kConf::get("partner_notification_name")</code>
+ 						</value>
+ 					</item>
+ 					<item objectType="KalturaEventNotificationParameter">
+ 						<key>entry_id</key>
+ 						<value objectType="KalturaEvalStringField">
+ 							<code>$scope->getEvent()->getObject()->getEntryId()</code>
+ 						</value>
+ 					</item>
+ 					<item objectType="KalturaEventNotificationParameter">
+ 						<key>entry_name</key>
+ 						<value objectType="KalturaEvalStringField">
+ 							<code>!is_null(entryPeer::retrieveByPk($scope->getEvent()->getObject()->getEntryId())) ? entryPeer::retrieveByPk($scope->getEvent()->getObject()->getEntryId())->getName() : ''</code>
```

Comment:
Please use $scope->getObject() to enable manual dispatching, also in entry_id parameter.